Modules/CTP/src/CTPTrendingTask.cxx

```diff
@@ -165,9 +174,16 @@ void CTPTrendingTask::generatePlots()
     if (auto histo = dynamic_cast<TH1*>(c->GetPrimitive("htemp"))) {
       if (index < 5)
         histo->SetTitle(mInputNames[index].c_str());
-      else
+      else if (index < 10)
```

Comment:
The guidelines recommend using curvy braces whether there is a single line or not.